contrib/init/upstart/REVIEWERS

```diff
@@ -1,2 +0,0 @@
-Tianon Gravi <admwiggin@gmail.com> (@tianon)
```

Comment:
Why not remove the entire upstart stuff?

Reply:
TBH; not sure if people are still using them 🤔 I recall when we stopped packaging it in our packages, that some people ran on non-systemd distros, and opened PR's to bring some of the things back; https://github.com/docker/docker-ce-packaging/blob/fcbe36668d1a66a2f7679ad191c9165c37c902e2/deb/common/docker-ce.docker.init#L49 (https://github.com/docker/docker-ce-packaging/pull/287).

I can have a look at removing them (and check if there's things in these directories that we missed in the docker-ce-packaging repo). Think that's better to do in a separate PR for visibility